### DISCUSSION TITLE
Create new menu

### DISCUSSION THREAD

**Discussion by HazemMohamed252** [statamic/cms#5453] [Q&A]
How can i create in cp new sidebar menu like content , fields , tools and under it sub menus. For example i want to create new MAIN sidebar call "Settings" and under it create "Main Settings" , "APP Settings" and anything like that. Please help.

**Comment by ryanmitchell** [comment]
You can see how I've done that here:
https://github.com/thoughtco/simple-commerce-nav/blob/5007f33ae426071f6335b929ab03e36f252495ea/src/ServiceProvider.php#L23

**Reply by HazemMohamed252** [reply]
After install it i got this error : Call to a member function showUrl() on null !!! Any help

**Comment by jasonvarga** [comment]
Check out the docs here: https://statamic.dev/extending/navigation

**Reply by HazemMohamed252** [reply]
I tried it but i can not reach for any result. all i want is to make new nav in the sidebar for example : Products and i can create blueprints for it and customize any fields
